.travis.yml

```diff
@@ -52,6 +52,10 @@ cache:
     - $HOME/gcloud/
     - $HOME/site-library
 before_install:
+- REPORT_PATH=${TRAVIS_BRANCH}_${TRAVIS_JOB_NUMBER};
+- if [ $TRAVIS_SECURE_ENV_VARS == true && $TRAVIS_EVENT_TYPE != cron ]; then
```

Comment:
It looks like you may have needed a second set of brackets. If you check the travis logs this line is currently erroring with the following message and failing to print the path correctly: 
`0.01s$ if [ $TRAVIS_SECURE_ENV_VARS == true && $TRAVIS_EVENT_TYPE != cron ]; then echo "Test report will be written to  https://storage.googleapis.com${HELLBENDER_TEST_LOGS}${REPORT_PATH}/tests/test/index.html"; fi
/home/travis/.travis/job_stages: line 57: [: missing `]'`